src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryAndroid.kt

```diff
@@ -3,19 +3,17 @@ package io.sentry.kotlin.multiplatform
 import android.content.Context
 import io.sentry.Sentry
 import io.sentry.android.core.SentryAndroid
+import io.sentry.android.core.SentryAndroidOptions
 
 internal actual object SentryBridge {
-
     actual fun captureMessage(message: String) {
         Sentry.captureMessage(message)
     }
 
-    actual fun start(dsn: String, context: Any?) {
-        SentryAndroid.init(context as Context) { options ->
-            options.dsn = dsn
-            options.isAttachStacktrace = true
-            options.isAttachThreads = true
-        }
+    actual fun start(context: Any?, configuration: OptionsConfiguration<SentryKMPOptions>) {
+        val options = SentryKMPOptions()
+        configuration.configure(options)
+        SentryAndroid.init(context as Context, convertToSentryAndroidOptions(options))
```

Comment:
We have to check if the instance is of the type Context first before calling `init`, otherwise it'd crash if its not.